composer.json

```diff
@@ -17,6 +17,7 @@
     "require": {
         "php": "^8.0",
         "ext-json": "*",
+        "ext-mbstring": "*",
```

Comment:
Not needed.

Reply:
We have  ext-mbstring installed.  It still fails.

Reply:
Please merge this.

Reply:
maybe it should be added in your composer.json

Reply:
@W0rma could you remove this line please, I also think it works without it.

Reply:
@eerison it should work without it.

Reply:
then removing it, IMO all threads are solved

Reply:
Isn't `mb_detect_encoding()` only available if the `mbstring` extension is installed?
That's the reason why I added this requirement.

Btw. codeception itself has such requirement as well ( https://github.com/Codeception/Codeception/blob/5.0/composer.json#L21 ).

Reply:
I don't know,  but isBinary was already using it for months and no one complains about it. And it is not explicitly in php docs 

https://www.php.net/manual/en/function.mb-detect-encoding.php

And have this requirement in codeception doesn't mean it is necessary in this lib.

Reply:
@W0rma  @eerison 
Yes, `mb_detect_encoding()` is part of the `mbstring` extension (https://www.php.net/manual/en/mbstring.installation.php). However, this module can't run without the main Codeception library, so it also uses Codeception dependencies. But we can leave it here for clarity.

Reply:
> Yes, mb_detect_encoding() is part of the mbstring extension

if it requires, then we should have this in composer.json

```json
"ext-mbstring": "*",
```

and @guraurobi we already `mb_detect_encoding` merged, then we should have this in composer.json, have https://github.com/Codeception/module-db/blob/185889230bccfc65d2c03624879ade5357ec43f9/src/Codeception/Lib/Driver/Db.php#L350

IMO **everything as clarified**, and the PR is **ready to merge**.